test/consumer_test.exs

```diff
@@ -49,10 +49,10 @@ defmodule NSQ.ConsumerTest do
     NSQ.Consumer.event_manager(consumer)
       |> GenEvent.add_handler(NSQ.ConsumerTest.EventForwarder, self())
 
-    HTTP.post("http://127.0.0.1:6751/put?topic=#{@test_topic}", [body: "hello"])
+    HTTP.post("http://127.0.0.1:6751/pub?topic=#{@test_topic}", [body: "hello"])
```

Comment:
/pub came with the V1 spec according to the changelog https://github.com/nsqio/nsq/blob/c22751c7e63cacb28e61c13b84efe95beb192910/ChangeLog.md#100-compat---2017-03-21
From what I can find in github, that was introduced on Jul 19, 2013
https://github.com/nsqio/nsq/commit/8b50fe86cdf1872282f1e3e80ade405c4d27d68a#diff-17367c04222666c193afbedd39b5fff1R26
and came with NSQ 0.2.2
https://github.com/nsqio/nsq/blob/master/ChangeLog.md#0222---2013-08-26